### DISCUSSION TITLE
Multiple Output Device, mute Built-in output (just a tip)

### DISCUSSION THREAD

**Discussion by luizen** [ExistentialAudio/BlackHole#102] [General]
Hi, just wanted to give a tip for those using an external audio interface, rather than the built-in one. 

As the [documentation](https://github.com/ExistentialAudio/BlackHole/wiki/Multi-Output-Device) says, when creating a _Multiple Output Device_, it is currently mandatory to enable the Built-in device.

However, if you don't want to have sound coming out of the built-in speaker, you can just open the `Audio Midi Setup->Audio Devices`, select the Built-in device on the left, and on the right check `Mute` option for the _Master_ channel on the _Master Stream_ panel.

This will make the sound come only out of your external audio interface and the Blackhole device.

**Comment by krono-i2** [comment]
Which macOS version are you using?
On El Capitan I’m able to remove built-in audio interface from the multi output device.

**Comment by devinroth** [comment]
It’s not that it can’t be removed but more that it needs to be there. There is a bug in macOS that requires the main device in a multi output to have 2 channels. Easiest solution is to include the build-in output and just mute it.

**Comment by krono-i2** [comment]
Ah ok... I’ve not such a bug on OS X El Capitan.

**Comment by devinroth** [comment]
Interesting. So it’s just in the newer versions. 

Devin

**Comment by luizen** [comment]
I am running High Sierra.

**Comment by smith12055** [comment]
Just what I was looking for. Thank you.
